index.js

```diff
@@ -1,13 +1,15 @@
 /* jshint node: true */
 'use strict';
 
+var env = require('./config/environment');
+
 module.exports = {
   name: 'ember-ag-grid',
 
   included: function(app) {
     this._super.included(app);
 
-    if (app.options.agGridEnterprise) {
+    if (typeof env().agGrid != "undefined" && env().agGrid.useEnterprise) {
```

Comment:
Please use strict equality here. Also, I think it would be cleaner to write it like this:
```js
var env = require('./config/environment')();

 module.exports = {
    name: 'ember-ag-grid',
  
    included: function(app) {
      this._super.included(app);
  
      if (typeof env.agGrid !== 'undefined' && env.agGrid.useEnterprise) {
```